packages/manager/apps/zimbra/src/routes/routes.tsx

```diff
@@ -178,6 +178,35 @@ export const Routes: any = [
             ...lazyRouteConfig(() =>
               import('@/pages/dashboard/Redirections/Redirections'),
             ),
+            children: [
+              {
+                path: 'add',
+                ...lazyRouteConfig(() =>
+                  import(
+                    '@/pages/dashboard/Redirections/AddAndEditRedirections.page'
+                  ),
+                ),
+                handle: { isOverridePage: true },
+              },
+              {
+                path: 'edit',
+                ...lazyRouteConfig(() =>
+                  import(
+                    '@/pages/dashboard/Redirections/AddAndEditRedirections.page'
+                  ),
+                ),
+                handle: { isOverridePage: true },
+              },
+              {
+                path: 'delete',
+                ...lazyRouteConfig(() =>
+                  import(
+                    '@/pages/dashboard/Redirections/AddAndEditRedirections.page'
+                  ),
+                ),
+                handle: { isOverridePage: true },
+              },
```

Comment:
No need for now or put a comment (todo: need change url when delete component will be available)